src/InputField.js

```diff
@@ -75,4 +64,21 @@ const InputField = ({ id, label, color, icon, info, ...props }) => {
   )
 }
 
+InputField.propTypes = {
+  /** Id passed to the native input element. Necessary to set up aria-describedby and for attributes for accessibility.  */
+  id: PropTypes.string.isRequired,
+  /** Text to be used as the label for the input element. */
+  label: PropTypes.string.isRequired,
+  /** The optional icon that shows up on the right side of the element. */
+  icon: PropTypes.string,
+  /** Optional border and icon color. */
+  color: PropTypes.string,
+  /** Additional content to go in the info drawer below the element. This will be replaced by a tooltip later probably.  */
+  info: PropTypes.string
+}
+
+InputField.defaultProps = {
```

Comment:
I don't *think* we need theme provided to composite components, do we? No big deal

Reply:
That should be true as long as there are no dependencies on `props.theme`. I've removed a few unnecessary default themes!